recipes/enterprise-extensions/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "enterprise_extensions" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ede50de0f26b6bd15cf209700465b511431bc04182db4b8d91b04f19a6be9e0a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    host:
+        - python >=3.6
+        - pip
+        - numpy >=1.16.3
+        - scipy >=1.2.0
+        - pytest
+
+    run:
+        - python >=3.6
+        - numpy >=1.16.3
+        - scipy >=1.2.0
+        - ephem >=3.7.6.0
+        - healpy >=1.14.0
+        - scikit-sparse >=0.4.5
+        - pint-pulsar >=0.8.2
+        - libstempo >=2.4.0
+        - enterprise-pulsar >=3.1.0
+        - emcee
+        - ptmcmcsampler
+        - numdifftools
```

Comment:
This is listed upstream in `requirements.txt`, but not in `install_requires` in `setup.py`, nor does it seem to be actually imported by the package?
```suggestion
```